packages/overlay/src/overlay-stack.ts

```diff
@@ -90,6 +90,11 @@ export class OverlayStack {
     }
 
     private startTabTrapping(): void {
+        if (!this.trappingInited) {
+            this.initTabTrapping();
```

Comment:
It might work a little better to have a `disableTabTrapping` option. I'd rather not cause all clients to do extra work if they're not running into the Safari issue.

Reply:
Other than the boolean check, I don't particularly count this as extra work. I had previously been making this call in the `constructor()` which means if an application doesn't use tab trapping this change now means they do less work.

A universal `disabledTabTrapping` options is interesting, however. Where do you think you'd want to apply that? Currently, there are no packages that leverage tab trapping on their own (IIRC), so an implementing app could also not use `type="modal"` (or manage their own usage of that value) and this would never be needed in that application. 

This does point to the benefit of a more durable solution. To prevent such a solution from adding work to non-Safari browsers, what are you thoughts on in component browser checking?

Reply:
Oh never mind -- I just misunderstood this logic. I'm not even sure how 🙈. This looks perfect.